hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/cache/CacheManager.java

```diff
@@ -60,18 +60,18 @@
     private static final String RESET_PUBLISH_CACHE_PROP = "hawkular-alerts.reset-publish-cache";
     private static final String RESET_PUBLISH_CACHE_ENV = "RESET_PUBLISH_CACHE";
 
-    private volatile boolean updateRequested = false;
-    private volatile boolean updating = false;
-
     @EJB
     PropertiesService properties;
 
     @EJB
     DefinitionsService definitions;
 
+    @Resource(lookup = "java:jboss/infinispan/cache/hawkular-alerts/dataIds")
+    private Cache<TriggerKey, Set<String>> dataIdsCache;
+
     // Note, Only the cache key is being used to today. The values are currently set to "" and reserved for future use.
```

Comment:
Please update this comment with the updated definition.  Maybe mention that the tenantId in the key is applicable to the value (so the values can just be a Set of triggerIds).